.appveyor.yml

```diff
@@ -42,20 +42,33 @@ before_test:
 
 test_script:
   - cd diesel
-  - cargo test --features "%backend% extras"
+  - cargo test --no-default-features --features "%backend% extras"
+  - cd ../diesel_derives
+  - cargo test --no-default-features --features "%backend%"
+  - cd ../diesel_infer_schema/infer_schema_internals
+  - cargo test --no-default-features --features "%backend%"
+  - cd ../../diesel_infer_schema/infer_schema_macros
+  - cargo test --no-default-features --features "%backend% dotenv"
+  - cd ../../diesel_infer_schema
+  - cargo test --no-default-features --features "%backend% dotenv_macro"
+  - cd ../diesel_migrations/migrations_internals
+  - cargo test
+  - cd ../../diesel_migrations/migrations_macros
+  - cargo test
+  - cd ../../diesel_migrations/
+  - cargo test --no-default-features --features "%backend%"
```

Comment:
Super unimportant and probably better in a separate PR: Instead of two lines per crate, you could also try to call this as `cargo test --no-default-features --features "%backend% extras" --manifest-path=diesel/Cargo.toml` (same is true for travis and this could make the files look more alike)